### PR TITLE
[Backport][ipa-4-9][manual] ipatests: Added test automation for SHA384withRSA CSR support

### PR DESCRIPTION
**Pull request by ssidhaye**
Scenario 1:
Setup master with --ca-signing-algorithm=SHA384withRSA
Run certutil and check Signing Algorithm

Scenario 2:
Setup a master
Stop services
Modify default.params.signingAlg in CS.cfg
Restart services
Resubmit cert (Resubmitted cert should have new Algorithm)

Pagure Link: https://pagure.io/freeipa/issue/8906

Signed-off-by: Sumedh Sidhaye <ssidhaye@redhat.com>

### CODE REVIEW

**Comment by flo-renaud** [comment]
@ssidhaye your ipa-4-9 branch looks a bit out of date, could you rebase to the latest ipa-4-9 version? It may also fix some of the issues in PRCI.

File: ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml [hunk 1]
```diff
@@ -588,6 +588,19 @@ jobs:
...
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-latest-ipa-4-9/build_url}'
+        selinux_enforcing: True
+        test_suite: test_integration/test_installation.py::TestInstallwithSHA384withRSA
+        template: *ci-ipa-4-9-latest
+        timeout: 3600
+        topology: *master_1repl_1client
```
Comment (flo-renaud) [inline ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml:602]: `*master_1repl`, no client is needed for the test

**Review by flo-renaud — COMMENTED** [review]
Hi @ssidhaye 
please find a small inline comment related to the test definition. Otherwise LGTM.

File: ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml (hunk 1 above)
Comment (ssidhaye) [reply ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml]: Fixed it

**Comment by flo-renaud** [comment]
@ssidhaye thanks for the update, ack

**Comment by flo-renaud** [comment]
ipa-4-9:

* 0edf915efbb39fac45c784171dd715ec6b28861a Added test automation for SHA384withRSA CSR support
* ef43ea03ef90cd34f2ce55a946df9a1d8e17badf Added nightly job definitions

**Pull request closed without merging**